internal/auth/assets/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>CalendarSync</title>
+</head>
+<body style='font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;'>
+    <div style="text-align: center; padding-top: 30px;">
+        <h2 style="color:#051c59; font-weight: 500; font-size: 30px; margin-bottom: 10px;">CalendarSync authentication successful!</h2>
+        <p style="font-size:20px; color:#7df481; margin-top: 10px;">You can now close this window.</p>
```

Comment:
While I like green in general, that green is a bit too bright in my opinion.

Reply:
That's what a colleague in the office today said as well. 

For the context: it's the "accent" color of the inovex color palette. 

I'll see what other options we have! 👀

Reply:
![image](https://github.com/inovex/CalendarSync/assets/13996238/715876be-5e68-4726-aca3-75a256a0142d)

Added another accent color, from the "blue color accent" palette. WDYT?

Reply:
Much better :-)